[14.0][ADD] web_widget_uom (#2091)
OCA/web: web_widget_uom/readme/DESCRIPTION.rst

This module aims to allow the user to decide how many
decimal places should be displayed in the Odoo User

> Interface for each Unit of Measure.

Reviewer: It should mention that `decimal.precision.digits` must be greater than `uom.uom.decimal_places`. Because if this is not the case it will be truncated nevertheless, or you can check it in Python.
A check in python is better IMO.

Author: @tarteo i've pushed some changes. 

Basically it now show min decimal places between the ones entered by user and decimal places set in uom. 

Could u check?

**edit**: 

I've slightly changed so that it shows decimal places set in UoM or, optionally, only user inputed decimals up to decimal places. In this way user can choose what's best use case.

> This module aims to allow the user to decide maximum
> decimal places should be displayed in the Odoo User
> Interface for each Unit of Measure.
> 
> If Show only inputed decimals is set in UoM,
> it shows inputed decimals up to Decimal Places.

Reviewer 2: ```suggestion
This module allows the user to decide maximum
how many decimal places should be displayed in the Odoo User
Interface for each Unit of Measure.

If "Show only inputed decimals" is enabled on the UoM,
the widget will show inputed decimals up to Decimal Places.
```
